README.md

```diff
@@ -133,3 +133,12 @@ For example:
 RAILS_ENV=production bin/rake remediate_collection['druid:hw105qf0103']`
 RAILS_ENV=production bin/rake remediate['druid:gq319xk9269','14373','shl','1']
 ```
+
+## WARC Listing
+
+If you would like to get a CSV of all the WARC files and their metadata (size,
```

Comment:
"If, for a collection, you would like to get ... "   I think may be a clearer read?